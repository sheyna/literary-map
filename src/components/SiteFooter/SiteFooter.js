import styles from './SiteFooter.module.css';

function SiteFooter() {
	return (
		<footer className={styles.SiteFooter}>
			<p>A project by Sheyna Watkins. Map icon images courtey <a href="https://www.davidrumsey.com/about/copyright-and-permissions" target="_blank" rel="noreferrer">David Rumsey Map Collection, David Rumsey Map Center, Stanford Libraries</a></p>
		</footer>
	)
}

export default SiteFooter;
