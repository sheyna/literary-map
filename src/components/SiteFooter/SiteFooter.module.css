.SiteFooter {
	background-color: #683257;
	color: #F2E9DF;
	padding: 1rem;
	bottom: 0;
	position: relative;
	flex-shrink: 0;
	margin-top: 4rem;
}

.SiteFooter p {
	margin: 0;
}

.SiteFooter a {
	color: #F2E9DF;
}
.SiteFooter a:visited {
	color: #F2E9DF;
}

@media screen and (min-width: 1025px) {
	.SiteFooter {
		margin-top: 3rem;
		width: 100%;
		padding: 1rem 6rem;
	}
}