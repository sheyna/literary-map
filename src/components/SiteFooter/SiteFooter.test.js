import ReactTestRenderer from 'react-test-renderer';
import SiteFooter from  './SiteFooter';

describe('SiteFooter component', () => {
  it('should match snapshot', () => {
    const component = ReactTestRenderer.create(
      <SiteFooter />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});