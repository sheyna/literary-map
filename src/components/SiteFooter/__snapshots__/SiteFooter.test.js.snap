// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteFooter component should match snapshot 1`] = `
<footer
  className="SiteFooter"
>
  <p>
    A project by Sheyna Watkins. Map icon images courtey 
    <a
      href="https://www.davidrumsey.com/about/copyright-and-permissions"
      rel="noreferrer"
      target="_blank"
    >
      David Rumsey Map Collection, David Rumsey Map Center, Stanford Libraries
    </a>
  </p>
</footer>
`;
